### ISSUE TITLE
VS Code set up - gulp tasks not recognized by VS code

### ISSUE THREAD

**Issue by kingjordan** [nvisionative/nvQuickTheme#8]
I had to run:
npm install
and npm gulp install
and then VS code would pick up the gulp tasks, might have to test this yourself

and possibly add to documentation

EDIT:
upon further testing,t he gukp taks will not run propery when triggered through the vs shortcut
Ctrl + p -> type 'task' -> 'enter a space' and the task list appears -> select task to run

get error:

> usr/bin/bash: node_modules.bingulp.cmd: command not found

However, if I run from the bash prompt, gulp task work

**Comment by tksheppard** [comment]
I installed VS Code today and setup a new instance of nvQuickTheme. After following install instructions, I was able to use gulp commands with the VS Shortcut codes. 

We recently had gone back and added and clarified that gulp needs to be installed on the global level for the commands to work correctly, which seems to be what the issue was here. Sorry for the inconvenience.

Thank you for your contribution!
